cluster/src/main/scala/io/snappydata/gemxd/ClusterCallbacksImpl.scala

```diff
@@ -183,7 +183,7 @@ object ClusterCallbacksImpl extends ClusterCallbacks with Logging {
     tablesArr.foreach(f = table => {
       Try {
         val tableData = session.sql(s"select * from $table;")
-        logDebug(s"EXPORT_DATA procedure is writing table: $table.")
+        logInfo(s"EXPORT_DATA procedure is exporting table: $table.")
```

Comment:
put more info if you want to log this. Like destination and format etc. Otherwise I don't see much value.

Reply:
The thought was to let user know which tables have been picked up for exporting, when ALL option is given instead of list of table names in the procedure args. Adding destination and format makes it better. Will make the changes. 
Thanks..